package.json

```
@@ -8,7 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
```

may i suggest we add linting to the test task?
```suggestion
    "test": "mocha && npm run lint",
```